### PR TITLE
Using the settings.logo_max_width value in styles

### PR DESCRIPTION
**Pull request by montalvomiguelo**
I added that value in the stylesheet in order to get this setting working.

screen shot 2015-08-01 at 9 29 25 pm

Address #460

### CODE REVIEW

**Comment by cshold** [comment]
Good catch. One adjustment that we've been using lately is to not require the `px` value in the text box. Seems insignificant, but turns out it's enough of a hassle to non-tech people that be worth it. Can you update this to:
```
{
  "type": "text",
  "id": "logo_max_width",
  "label": "Max logo width",
  "info": "Defined in pixels. Do not add the 'px' unit."
}
```

**Comment by imontalvomiguel** [comment]
Yahh You right... I have updated the files :)

File: assets/timber.scss.liquid [hunk 1]
```diff
@@ -113,6 +113,9 @@ $colorNavText: #333;
 $colorFooterBg: {{ settings.color_footer_bg }};
 $colorFooterText: {{ settings.color_footer_text }};
 
+// Logo max width
+$logoMaxWidth: {{ settings.logo_max_width | default: '450' }}px;
```
Comment (bakura10) [inline assets/timber.scss.liquid]: I personally do this in the theme, so even for people including the px, it does not break:

{{ settings.logo_max_width | default: '450' | remove: 'px' }}px;

Comment (cshold) [inline assets/timber.scss.liquid]: That is smart! @montalvomiguelo are you able to adjust that line and we can merge it in.

Comment (montalvomiguelo) [inline assets/timber.scss.liquid]: Sure

**Pull request merged**